nautobot/docs/user-guide/feature-guides/wireless-networks-and-controllers.md

```diff
@@ -0,0 +1,118 @@
+# Wireless Networks and Controllers
+
+This guide demonstrates how to set up Wireless Networks and Controllers to manage Wireless Access Points via a Controller in a Data Center. In this example, we use **Central Mode**, where traffic from the Access Points is tunneled through the Controller.
+
+![Central Mode Diagram](images/wireless/central-mode.png)
+
+## Optional Prerequisites
```

Comment:
I don't have a better suggestion but "Optional Prerequisites" seems to contradict itself. Maybe "Optional Preparation" or "Optional Setup"? Might be fine as-is.